R/appStatic.R

```diff
@@ -6,18 +6,43 @@
 #' @param logo Name of a logo or path to a logo. If NULL no logo is included.
 #' Only svg format allowed for the moment.
 #' @param title title of the shiny
+#' @param background Content to fill the background panel. If `NULL`, this panel
+#' will not appear in the Shiny UI. This argument can be populated in two ways:
+#' 1) A character string containing `bslib` code to go directly inside
+#' `bslib::card()`.
+#' 2) A named character vector where the names correspond to specific`bslib`
+#' card elements: "header", "title", "subtitle", "body", and "footer". Each
+#' element's name indicates the type of text it will fill. The order of these
+#' elements is important when displaying the content. Markdown sythax for text
+#' styling is supported.
 #' @param open Whether to open the shiny app project.
 #'
 #' @return The shiny app will be created in directory.
+#'
 #' @export
 #'
+#' @examples {
```

Comment:
maybe this content would fit more for a vignette than as an example?

Reply:
Do I eliminate these examples¿

Reply:
Maybe you can copy that in a simple vignette (so we do not lose what you've already done) and when we have a stable behaviour for background argument we should write a proper vignette @nmercadeb